eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -100,46 +100,20 @@ jobs:
 
         - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 
-        - task:  UniversalPackages@0
+        - pwsh: azcopy copy "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/*" "$(System.DefaultWorkingDirectory)/parser" --recursive
```

Comment:
We will want to build in the version in the path so we can get the exact version.